setup.cfg

```diff
@@ -1,3 +1,3 @@
 [pycodestyle]
-ignore = E402
-max-line-length = 80
+ignore = E121, E123, E126, E226, E24, E704, W503, E402
+max-line-length = 120
```

Comment:
I agree that most of the times keeping it under 80 is fine and easily doable, but sometimes it just makes things less readable. These exceptions are typically `if` statements where some logic is involved and it doesn't help that with python we have to use 4 spaces for each indentation.
But ok, I can live with 80 and it's not worth my time debating about this.